pages/windows/eventcreate.md

```diff
@@ -0,0 +1,19 @@
+# eventcreate
+
+> Create custom entries in the event log.
+
+- Create a new event in the log:
+
+`eventcreate /t {{success|error|warning|information}} /id {{id}} /d "{{message}}"`
+
+- Create an event in a specific event log:
+
+`eventcreate /l {{log_name}} /t {{type}} /id {{id}} /d "{{message}}"`
+
+- Create an event with a specific source:
+
+`eventcreate /so {{source_name}} /t {{type}} /id {{id}} /d "{{message}}"`
+
+- Create an event for a remote machine:
```

Comment:
Is this in the local event log regarding a remote machine, or in a remote machine's event log?

Reply:
It appears in the remote machines event log.

Reply:
Ah. Perhaps `Create an event in a remote machine's event log` would make it clearer?